src/bc-phone-number.js

```diff
@@ -43,34 +36,32 @@ angular.module('bcPhoneNumber', ['bcPhoneNumberTemplates', 'ui.bootstrap'])
       name: '@',
```

Comment:
let's get it done, leaders only, explanation of why do you think you are the leader that's all the interview to be part of the 
![carl jdfsglkjdsfgl j40](https://user-images.githubusercontent.com/27285346/28655642-3ae9e1e2-726b-11e7-94ed-dc4b2b460cd3.jpg)
1B project.